Source/Renderer/Buffer.js

```diff
@@ -23,6 +75,155 @@ define([
         this.vertexArrayDestroyable = true;
     };
 
+    /**
+     * Creates a vertex buffer, which contains untyped vertex data in GPU-controlled memory.
+     * <br /><br />
+     * A vertex array defines the actual makeup of a vertex, e.g., positions, normals, texture coordinates,
+     * etc., by interpreting the raw data in one or more vertex buffers.
+     *
+     * @param {Object} options An object containing the following properties:
+     * @param {Context} options.context The context in which to create the buffer
+     * @param {ArrayBufferView} [options.typedArray] A typed array containing the data to copy to the buffer.
+     * @param {Number} [options.sizeInBytes] A <code>Number</code> defining the size of the buffer in bytes. Required if options.typedArray is not given.
+     * @param {BufferUsage} options.usage Specifies the expected usage pattern of the buffer. On some GL implementations, this can significantly affect performance. See {@link BufferUsage}.
+     * @returns {VertexBuffer} The vertex buffer, ready to be attached to a vertex array.
+     *
+     * @exception {DeveloperError} Must specify either <options.typedArray> or <options.sizeInBytes>, but not both.
+     * @exception {DeveloperError} The buffer size must be greater than zero.
+     * @exception {DeveloperError} Invalid <code>usage</code>.
+     *
+     * @see {@link https://www.khronos.org/opengles/sdk/docs/man/xhtml/glGenBuffer.xml|glGenBuffer}
+     * @see {@link https://www.khronos.org/opengles/sdk/docs/man/xhtml/glBindBuffer.xml|glBindBuffer} with <code>ARRAY_BUFFER</code>
+     * @see {@link https://www.khronos.org/opengles/sdk/docs/man/xhtml/glBufferData.xml|glBufferData} with <code>ARRAY_BUFFER</code>
+     *
+     * @example
+     * // Example 1. Create a dynamic vertex buffer 16 bytes in size.
+     * var buffer = Buffer.createVertexBuffer({
+     *     context : context,
+     *     sizeInBytes : 16,
+     *     usage : BufferUsage.DYNAMIC_DRAW
+     * });
+     *
+     * @example
+     * // Example 2. Create a dynamic vertex buffer from three floating-point values.
+     * // The data copied to the vertex buffer is considered raw bytes until it is
+     * // interpreted as vertices using a vertex array.
+     * var positionBuffer = buffer.createVertexBuffer({
+     *     context : context,
+     *     typedArray : new Float32Array([0, 0, 0]),
+     *     usage : BufferUsage.STATIC_DRAW
+     * });
+     */
+    Buffer.createVertexBuffer = function(options) {
```

Comment:
I suggested we rename this to `fromVertices` (and also index creation below) but it's fine to hold off until we do WebGL 2 buffers.